test/lib/graphql/schema_comparator/diff/schema_test.rb

```diff
@@ -168,6 +168,7 @@ def test_changes_kitchensink
       "Deprecation reason on field `CType.a` has changed from `whynot` to `cuz`",
       "Argument `arg: Int` added to field `CType.a`",
       "Default value `10` was added to argument `arg` on field `CType.d`",
+      "Default value for argument `anotherArg` on directive `yolo` changed from `__no_default__` to `Test`",
```

Comment:
@eapache does this make sense?

Reply:
The gem definitely uses `__no_default__` as an internal placeholder for no default value. Arguably the schema comparator should handle that more gracefully, but I think this is ok enough.